pkg/api/client.go

```diff
@@ -78,13 +83,13 @@ func NewClient(cfg config.TLSConfig, proxyURL string) Client {
 }
 
 func getTimeoutFromEnvironment() time.Duration {
-	cfgHTTPClientTimeout := os.Getenv("HTTP_CLIENT_TIMEOUT")
+	cfgHTTPClientTimeout := os.Getenv("CENTRAL_CLIENTTIMEOUT")
```

Comment:
Was this supposed to change? CENTRAL_CLIENTTIMEOUT is for central specifically to override what the HTTP_CLIENT_TIMEOUT might have

Reply:
following...

Reply:
resolved